File: c#/producto.cs
using System.Collections.Generic;
public class producto{
	private int _idProducto;
	private string _nombreProducto;
	private List<ingrediente> _receta;
	private float _precio;

	public producto(){}

	public producto(int _idProducto, string _nombreProducto, List<ingrediente> _receta, float _precio){
		this._idProducto = _idProducto;
		this._nombreProducto = _nombreProducto;
		this._receta = _receta;
		this._precio = _precio;
	}

	public int IdProducto{
		get{
			return _idProducto;
		}
		set{
			_idProducto = value;
		}
	}

	public string NombreProducto{
		get{
			return _nombreProducto;
		}
		set{
			_nombreProducto = value;
		}
	}

	public List<ingrediente> Receta{
		get{
			return _receta;
		}
		set{
			_receta = value;
		}
	}

	public float Precio{
		get{
			return _precio;
		}
		set{
			_precio = value;
		}
	}
}

File: java/Producto.java
import java.util.ArrayList;
public class Producto{
	private int idProducto;
	private String nombProducto;
	private ArrayList<Ingrediente> Receta;
	private float precio;
	
	public Producto(){
		Receta=new ArrayList<Ingrediente>();
	}
	public Producto(int idProducto,String nombProducto,float precio){
		this.idProducto=idProducto;
		this.nombProducto=nombProducto;
		this.precio=precio;
		Receta=new ArrayList<Ingrediente>();
	}
	
	public void setidProducto(int idProducto){
		this.idProducto=idProducto;
	}
	public int getidProducto(){
		return this.idProducto;
	}
	
	public void setnombProducto(String nombProducto){
		this.nombProducto=nombProducto;
	}
	public String getnombProducto(){
		return this.nombProducto;
	}
	
	public void setReceta(ArrayList<Ingrediente> Receta){
		this.Receta=Receta;
	}
	public ArrayList<Ingrediente> getReceta(){
		return this.Receta;
	}	
	
	public void setprecio(float precio){
		this.precio=precio;
	}
	public float getprecio(){
		return this.precio;
	}
	
}